### PR TITLE
chore(create-playwright): enhance UX follow-ups and add more verbose Playwright Test config

### PR DESCRIPTION
**Pull request by mxschmitt**
- [x] Update JS examples + config

Relates #8892

### CODE REVIEW

File: packages/create-playwright/assets/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (mxschmitt) [inline packages/create-playwright/assets/.eslintrc.js:1]: drive-by: migrated instead of ignoring the assets I only disable the notice rule.

File: packages/create-playwright/assets/playwright.config.ts [hunk 2]
```diff
@@ -3,8 +3,14 @@ import path from 'path';
 
 // Reference: https://playwright.dev/docs/test-configuration
 const config: PlaywrightTestConfig = {
+  // Timeout per test
   timeout: 30 * 1000,
+  // Test directory
   testDir: path.join(__dirname, '{{testDir}}'),
+  // If a test fails, retry it additional 2 times until it get marked as failing.
+  retries: 2,
+  // Artifacts folder where screenshots, videos, and traces are stored. Usually this gets uploaded on your CI artifacts store.
```
Comment (dgozman) [inline packages/create-playwright/assets/playwright.config.ts:12]: Drop the "usually this gets ...." part.

File: packages/create-playwright/assets/example.spec.ts [hunk 3]
```diff
@@ -1,7 +1,89 @@
-import { test, expect } from '@playwright/test';
+import { expect, Page, test as base } from '@playwright/test';
```
Comment (dgozman) [inline packages/create-playwright/assets/example.spec.ts:1]: - What happens to example.spec.js?
- I don't think this works as a "get started example". I'd rather make "1-get-started", "2-recipes", "3-pom" directories with examples or something like that.

Comment (mxschmitt) [reply packages/create-playwright/assets/example.spec.ts]: Will follow-up in more PRs dedicated to example test files.

File: packages/create-playwright/src/index.ts [hunk 4]
```diff
@@ -171,6 +172,10 @@ We suggest that you begin by typing:
 
 ${colors.cyan(prefix + '  ' + commandToRunTests(this.packageManager))}
 
+And check out the following files:
+  - ./${pathToNavigate ? pathToNavigate + '/' : ''}${exampleSpecPath} - Example end-to-end test of a TODO application
```
Comment (dgozman) [inline packages/create-playwright/src/index.ts:176]: This is not a test of a TODO application anymore :)

**Review by dgozman — APPROVED** [review]
